### DISCUSSION TITLE
TOTP doesn't work in latest v4.38.0-beta3 build

### DISCUSSION THREAD

**Discussion by ghost** [authelia/authelia#6448] [Pre-Release Feedback]
Image in use:

```
REPOSITORY                             TAG                      IMAGE ID       CREATED         SIZE
ghcr.io/authelia/authelia              v4.38.0-beta3            4630d2f9b63c   3 days ago      59.7MB
```

Latest beta build seems to be have broken TOTP functionality. @Dulanic can probably chime in on this as he was having issues as well.

If I try to login with TOTP:

> time="2023-12-23T22:59:01-06:00" level=error msg="Error occurred validating a TOTP authentication for user 'REDACTED': error occurred checking the TOTP history" error="error checking if TOTP history exists: expected 2 arguments, got 3" method=POST path=/api/secondfactor/totp remote_ip=172.20.69.50 stack="github.com/authelia/authelia/v4/internal/handlers/handler_sign_totp.go:139 TimeBasedOneTimePasswordPOST\ngithub.com/authelia/authelia/v4/internal/middlewares/require_auth.go:19    Require1FA.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/bridge.go:54          handleRouter.(*BridgeBuilder).Build.func6.1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:56         SecurityHeadersCSPNone.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:82         SecurityHeadersNoStore.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:12         SecurityHeaders.func1\ngithub.com/fasthttp/router@v1.4.22/router.go:420                           (*Router).Handler\ngithub.com/authelia/authelia/v4/internal/middlewares/log_request.go:14     handleRouter.LogRequest.func40\ngithub.com/valyala/fasthttp@v1.51.0/server.go:2359                         (*Server).serveConn\ngithub.com/valyala/fasthttp@v1.51.0/workerpool.go:224                      (*workerPool).workerFunc\ngithub.com/valyala/fasthttp@v1.51.0/workerpool.go:196                      (*workerPool).getCh.func1\nruntime/asm_amd64.s:1650                                                   goexit"

**Comment by james-d-elliott** [comment]
Which storage provider?

**Reply by ghost** [reply]
`postgres:16-alpine`

**Reply by james-d-elliott** [reply]
Latest push should have fixed it.

**Reply by ghost** [reply]
That got it. Thanks. :)

**Reply by ghost** [reply]
Edit: Opened https://github.com/authelia/authelia/discussions/6457 instead.
